components/Text/Text.js

```diff
@@ -3,36 +3,34 @@ import * as styles from "./Text.module.scss";
 export default function Text({ mode, style, children }) {
   function styleClass() {
     switch (style) {
```

Comment:
```suggestion
  function styleClass() {
    const baseClass = `text`;
    const sizeClass = `text--${size}`;
    const headingClass = `text--${heading}`;
    const weightClass = `text--${weight}`;

    return [styles[baseClass], styles[sizeClass], styles[headingClass], styles[weigthClass]].join(' ');
```
what do you think on doing that way? is easier than the switch